testdata/test1.esdl

```diff
@@ -26,7 +26,7 @@
         <port xsi:type="esdl:InPort" connectedTo="935fb733-9f76-4a8d-8899-1ad8689a4b12" id="9c258b9d-3149-4720-8931-f4bef1080ec1" carrier="0bd9cb08-2f69-4e97-8ac8-bd87b07e466a_ret" name="In"/>
         <geometry xsi:type="esdl:Point" CRS="WGS84" lon="4.558639526367188" lat="52.148869383489114"/>
       </asset>
-      <asset xsi:type="esdl:Pipe" related="Pipe1_ret" id="Pipe1" length="6267.0" name="Pipe1" innerDiameter="0.5">
+      <asset xsi:type="esdl:Pipe" related="Pipe1_ret" id="Pipe1" length="6267.0" name="# type: ignore1" innerDiameter="0.5">
```

Comment:
This seems like an odd change,right? Why would you have a name='# type: ignore1"? Did you perform an automatic change?

Reply:
This should have been by mistake. I fixed it.